src/Account.sol

```diff
@@ -436,19 +436,15 @@ contract Account is IAccount, Auth, OpsReady {
             });
         } else if (_amount < 0) {
             // if amount is negative, withdraw
-            if (_abs(_amount) > freeMargin()) {
-                /// @dev make sure committed margin isn't withdrawn
-                revert InsufficientFreeMargin(freeMargin(), _abs(_amount));
-            } else {
-                /// @dev failed Synthetix asset transfer will revert and not return false if unsuccessful
-                MARGIN_ASSET.transfer(owner, _abs(_amount));
+            _sufficientMargin({_marginOut: _amount});
```

Comment:
Seems a bit excessive to use named params for single arguments.

Reply:
agreed. ngl this was copilot copying my style lol

Reply:
removed